src/main/java/org/candlepin/model/ConsumerCurator.java

```diff
@@ -178,19 +178,32 @@ public Map<String, Collection<String>> getFacts() {
         }
     }
 
-    @Inject private EntitlementCurator entitlementCurator;
-    @Inject private ConsumerTypeCurator consumerTypeCurator;
-    @Inject private ContentAccessCertificateCurator contentAccessCertificateCurator;
-    @Inject private CertificateSerialCurator certificateSerialCurator;
-    @Inject private DeletedConsumerCurator deletedConsumerCurator;
-    @Inject private FactValidator factValidator;
-    @Inject private IdentityCertificateCurator identityCertificateCurator;
-
-    @Inject private Provider<HostCache> cachedHostsProvider;
-    @Inject private PrincipalProvider principalProvider;
-
-    public ConsumerCurator() {
+    private EntitlementCurator entitlementCurator;
+    private ConsumerTypeCurator consumerTypeCurator;
+    private ContentAccessCertificateCurator contentAccessCertificateCurator;
+    private CertificateSerialCurator certificateSerialCurator;
+    private DeletedConsumerCurator deletedConsumerCurator;
+    private FactValidator factValidator;
+    private IdentityCertificateCurator identityCertificateCurator;
```

Comment:
Sonarqube is identifying this and a couple other values as unused and safe for removal. Given that we're updating the constructor's input, this seems like a good time to remove them entirely.